integration/custom_aggregation_test.go

```diff
@@ -26,16 +26,17 @@ import (
 	"testing"
 	"time"
 
+	"github.com/m3db/m3metrics/aggregation"
 	"github.com/m3db/m3metrics/policy"
 	xtime "github.com/m3db/m3x/time"
 
 	"github.com/stretchr/testify/require"
 )
 
 var (
-	compressor                             = policy.NewAggregationIDCompressor()
-	compressedMin, _                       = compressor.Compress(policy.AggregationTypes{policy.Min})
-	compressedMinAndMax, _                 = compressor.Compress(policy.AggregationTypes{policy.Min, policy.Max})
+	compressor                             = aggregation.NewIDCompressor()
+	compressedMin, _                       = compressor.Compress(aggregation.Types{aggregation.Min})
```

Comment:
Can you change this and the following one to use `MustCompress`? Thanks

Reply:
sounds good